Changes

```diff
@@ -1,5 +1,14 @@
 Release history for Zonemaster component Zonemaster-LDNS
 
+2.3.0 2022-06-09
```

Comment:
I understand from the previous conversation that this release is 2.2.2. Could you update it?

Reply:
I do not think it is very important if it is 2.2.2 or 2.3.0 but I think that 2.3.0 is more in line with what we have done before. I'll downgrade to 2.2.2 since there seems to be three in favor of that.